io/ilvis2/Ilvis2Reader.cpp

```diff
@@ -50,14 +50,16 @@ std::string Ilvis2Reader::getName() const { return s_info.name; }
 Options Ilvis2Reader::getDefaultOptions()
 {
     Options options;
-    Option mapping("mapping", "LOW");
-    options.add(mapping);
+    options.add("mapping", "ALL", "The point to extract from the shot: LOW, HIGH, or ALL.  ALL creates 1 or 2 points per shot, depending on if LOW and HIGH are the same.");
```

Comment:
I can make that change.  I missed that LASWriter/LASReader was using a toupper call on the fixed param it uses (compression), so I'll implement that.

I will also modify the code to internally use an ENUM, I agree it's cleaner (and slightly faster internally than using string comparisons on every iteration of the point view).